Develop

KabohaJeanMark opened the pull request:
**Add media queries to change and adapt layout on different pages.**-

- Apply styling for phone, tablet and laptop screen sizes.
- Use fluid measurements such as percentages for width and vh and vw.
- Use flexbox and change layout for different screen sizes

KabohaJeanMark committed:
Update README.md

- Add instructions, screenshot image and links. Follow Professional GitHub repo setup
- Add link to MIT license

KabohaJeanMark committed:
Setup project following best practices

- Add README.md with guielines of Professional GitHub repo.
- Link README.md to MIT license
- Setup and Install Linters; Lighthouse, Webhint and Styelint

KabohaJeanMark committed:
Add HTML for page layout

- Add 4 sections of webpage that are to be implemented
- Use semantic HTML5 tags

KabohaJeanMark committed:
Add styling for Header section

- Use flexbox to setup styling for nav
- use font awesome for search and hamburger menu icons. Style them.
- Style with Mobile first approach

KabohaJeanMark committed:
Add styling to footer

- Use flexbos
- Use fluid measurements for responsiveness
- Use absolute positioning to place the footer at the bottom
- Use font awesome icons for social media links

KabohaJeanMark committed:
Add Styling for Highlight Section

- Add background image with linear gradient
- Add absolute positioned text ontop
- Used flexbox column direction

KabohaJeanMark committed:
Merge pull request #1 from KabohaJeanMark/ft-homepage

Add structure and default styling to homepage

tanzila-abedin committed:
add funding structure and styling

tanzila-abedin committed:
clear web and style linter error

tanzila-abedin committed:
fix more styling lint errors

tanzila-abedin committed:
fix hint error

KabohaJeanMark committed:
Merge pull request #2 from KabohaJeanMark/ft-tanzila

Add funding structure and styling

KabohaJeanMark committed:
Add media queries for tablet

- Add responsive styling for funding section.
- Style media queries for footer

KabohaJeanMark committed:
Add media queries to enforce responsiveness on different screen sizes

- add media queries for phone and tablet
- do separate styling based on media queries and change in layout
- increase font size on bigger screens and increase padding for mobile touch

KabohaJeanMark committed:
Merge pull request #3 from KabohaJeanMark/ft-media-queries-tablet

Add media queries for table and laptop

KabohaJeanMark committed:
Update README.md

- Add screenshots of different page sizes and responsiveness.
- Add social media contact info of my coding partner

KabohaJeanMark committed:
Merge pull request #5 from KabohaJeanMark/ft-update-readme

Update README.md

KabohaJeanMark committed:
Update inspiration

Add the link to the original site being cloned.

KabohaJeanMark committed:
Upddate live link

- Us gh pages

mikenath223 reviewed (changes requested):
### Hello Team 👋

Good job here you've done pretty well using semantic HTML.

However, there are a few issues to be fixed.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

**README**
- [ ] The live link doesn't work kindly fix it.

### **HEADER**

- [ ] On the desktop screen there are nav links on the header please include them and remove the hamburger menu on the desktop screen.
- [ ] Please add the TNW logo at the left top corner of the header as it appears on the original.
- [ ] Also, on all screen sizes ensure that it sticks to the top when the user scrolls.
- [ ] The hamburger menu on the header should only appear on mobile and tablet screens.
- [ ] Also on mobile and tablet screens the TNW logo appears a the top left corner of the header, please include it.

**### Highlights (top 3 articles)**

On the desktop screen size for the Highlights section please take note of the following three points and fix them:
* [ ] The TWN logo appears over the first article, at the top left corner
* [ ] There is a red overlay with a gradient in the top half of the first article
* [ ] The date appears over the first article, at the top right corner

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

KabohaJeanMark committed:
Work on feedback from PR review

- Added live link
- Add logo with red gradient
- Hide hamburger menu
- Make items mre responsive by changing font sizes

KabohaJeanMark committed:
Add images

KabohaJeanMark committed:
Merge pull request #7 from KabohaJeanMark/ch-refactor-code-feedback

Work on feedback from PR review

tanzila-abedin commented:
Hello 
Thank you for the feedback.
We have made the changes you asked for. Kindly have a look.

KabohaJeanMark committed:
Modify large screen media query

andresporras3423 reviewed (changes requested):
### Status: Changes Required :recycle:
**Hello Tanzila Abedin & Marcus Kaboha** 
Good job with the improvements for this project :muscle:  :muscle:
Your page look similar to the original in the desktop screen size :clap: :clap: :clap:  
There are just a few details to improve before you can go ahead:

- [ ] Yellow section (close to the top) most have a bigger bottom and top padding
- [ ] For the mobile version, it should appear the logo in the leftmost part of your navbar
- [ ] For the mobile version, it should appear a burger icon in the rightmost part of your navbar
- [ ] Use the same breakpoints of the original
- [ ] About the three top articles, use "hx" (h1,h2...) tag and "a" tag for the title of these articles
- [ ] use "hx" (h1,h2...) tag and "a" tag for the title of this section
- [ ] For each item in the lastest funding section: use "a" tag for the bottom text, use "a" tag for the two items listed, and use "hx" (h1,h2...) tag and "a" tag for the title of this items.
- [ ] In the footer, all the social media are centered using nav tag
- [ ] In the footer, the links below the social media are centered using a nav tag
 
Please request another review once you are ready! :+1: 
 
Happy coding :relaxed:
 
Oscar Andrés Russi Porras

KabohaJeanMark committed:
Add Styling for Big and Small laptop media queries to differentiate

- Add styling for big laptop size
- Use fixed width with absolute positioned image

KabohaJeanMark committed:
Merge pull request #8 from KabohaJeanMark/bg-large-small-laptop-media-query

Add large laptop media query

tanzila-abedin committed:
resize yellow header for laptop view

tanzila-abedin committed:
Merge pull request #9 from KabohaJeanMark/feedback-2

Changes on Feedback 2

tanzila-abedin commented:
Thank you for your feedback @andresporras3423 .

We have worked on resizing and restructuring the yellow nav-bar according to the project scope. Most of the requested changes have been implemented.
However, the rest of the changes are according to the live site but not according to specifications given by  Microverse for this project. 
Thank you.
![project-instructions](https://user-images.githubusercontent.com/60758407/109025084-15b9ce00-76c7-11eb-9e32-62d312c23547.png)
![project-requirements](https://user-images.githubusercontent.com/60758407/109025093-17839180-76c7-11eb-8120-ee5c5cb1674a.png)

tanzila-abedin commented:
Hello, we have made the necessary changes. kindly review our project.

teekaytech commented on index.html:
```diff
@@ -0,0 +1,204 @@
+                    <p>Tuesday - January 26, 2021</p>
+                </div>
+                <div class="lower-moon">
+                    Opal cuts of your apps' internet access to help you stop doomscrolling
```
In the Highlights (top 3 articles) section, 
- The title of each article should use a header (h1...h6) tag and has a link (a tag). Kindly make the changes. i.e.
```html
<h1>
  <a href="#">
    Opal cuts of your apps' internet access to help you stop doomscrolling
  </a>
</h1>
```

teekaytech commented on index.html:
```diff
@@ -0,0 +1,204 @@
+        <div class="latest-funding">
+            <div class="funding">
+                <div class="funding-header">
+                    <a href="#">Latest Funding Rounds On Index.Co <i class="fa fa-angle-right" aria-hidden="true"></i></a>
```
The title of this section should use a header (h1...h6) tag. i.e. 
```
<h3>
  <a href="#">
    Latest Funding Rounds On Index.Co 
    <i class="fa fa-angle-right" aria-hidden="true"></i>
  </a>
</h3>
```

KabohaJeanMark replied:
Thanks for clearly explaining and making a suggestion. This has been worked on by us.

teekaytech commented on index.html:
```diff
@@ -0,0 +1,204 @@
+                <div class="funding-header">
+                    <a href="#">Latest Funding Rounds On Index.Co <i class="fa fa-angle-right" aria-hidden="true"></i></a>
+                </div>
+                <div class="funding-links">
+                    <a href="#">See More</a>
+                    <a href="#">Get updates By Email</a>
+                    <a href="#" class="third-funding-link">Add info</a>
+                </div>
+
```
The nav menu (See more, Get updates, etc.) are grouped in a `<nav>` tag, not a `<div>` tag.

KabohaJeanMark replied:
Thanks. All link groupings have been grouped in nav instead of div tags

teekaytech commented on index.html:
```diff
@@ -0,0 +1,204 @@
+                    </div>
+
+                    <div class="three">
+                        <p><i class="fa fa-tag" aria-hidden="true"></i> Telecommunications</p>
+                        <p><i class="fa fa-briefcase" aria-hidden="true"></i> Expeditions Growth Capital</p>
```
These items should be linked (i.e. wrapped inside the `<a>` tag) for all the articles in this section.

KabohaJeanMark replied:
Done. Thanks for clear feedback.

teekaytech commented on index.html:
```diff
@@ -0,0 +1,204 @@
+        </div>
+    </main>
+    <footer>
+        <div class="social-media">
+            <a href="#"><i class="fab fa-facebook-square"></i></a>
+            <a href="#"><i class="fab fa-twitter"></i></a>
+            <a href="#"><i class="fab fa-youtube"></i></a>
+            <a href="#"><i class="fab fa-instagram"></i></a>
+            <a href="#"><i class="fab fa-pinterest"></i></a>
+            <a href="#"><i class="fas fa-rss"></i></a>
+            <a href="#"><i class="fas fa-envelope"></i></a>
+        </div>
```
These social media icons should be centered in a `nav` tag, not a `div` tag.

teekaytech commented on index.html:
```diff
@@ -0,0 +1,204 @@
+            <a href="#"><i class="fas fa-rss"></i></a>
+            <a href="#"><i class="fas fa-envelope"></i></a>
+        </div>
+        <div class="footer-links">
+            <a href="#">EVENTS</a>
+            <a href="#">ABOUT</a>
+            <a href="#">ADVERTISE</a>
+            <a href="#">JOBS</a>
+            <a href="#">CONTACT</a>
+        </div>
```
These menu items should also be centered in a `nav` tag, not a `div` tag.

teekaytech reviewed (changes requested):
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__Thank you for making most of the required changes highlighted by the previous reviewers :handshake:__

- Your responsiveness is so sweet :+1: 
- You have a clean and readable codebase :100: 

__However, some changes are required before this milestone can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [ ] Do not commit files that are not related to the project (e.g. `.vscode`). Instead, add them to [the `.gitignore` file](https://help.github.com/en/github/using-git/ignoring-files#create-a-local-gitignore).
- [ ] The remaining changes are highlighted in the inline comments below. 
- Most of these changes are semantically related. 
- It is advisable to use [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) tags so that your project will be meaningful to the browser, and not just the viewers.


* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

KabohaJeanMark committed:
Make code more semantic

- Add h1 semantic header tags for each article and link in anchor tag
- Use h3 instead of div for class funding-header
- Group funding links in a nav tag instead of div

KabohaJeanMark committed:
Work on feedback 3

- Sort out styling after grouping links in navs
- Wrap the paragraphs below funding in anchor tags to link them

KabohaJeanMark committed:
Use semantic nav tags for links on footer

- Replace div tags with nav tags for grouping the links on the footer.
- Replaes for both social media and page links

KabohaJeanMark committed:
Finishe feedback 3

- DOnt commit .vscode. Update .gitignore to include it and prevent pushing.
- Use proper HTML5 semantic tags (h1,h2..) for headings and nav for link groups
- Improve padding

KabohaJeanMark committed:
Merge pull request #10 from KabohaJeanMark/ft-feedback-3

Use HTML5 semantic tags

KabohaJeanMark commented:
Hello :wave: @teekaytech 

Thanks for the clear feedback for improvements.
We have worked on all of it  using the proper semantic HTML tags for the headings and the nav tags for those link groupings 💪🏾

tanzila-abedin commented:
Hello wave @teekaytech

Thanks for the clear feedback for improvements.
We have worked on all of it using the proper semantic HTML tags for the headings and the nav tags for those link groupings 💪🏾
